Client/Frontend/Browser/Search/OpenSearch.swift

```diff
@@ -121,32 +128,47 @@ class OpenSearchEngine: NSObject, NSSecureCoding {
     /**
      * Returns the search suggestion URL for the given query.
      */
-    func suggestURLForQuery(_ query: String) -> URL? {
+    func suggestURLForQuery(_ query: String, locale: Locale = Locale.current) -> URL? {
```

Comment:
I'm not sure adding a new parameter is necessary here, now, since we never pass anything in.

Reply:
As with previous comment, this is good for unit test purposes.
Atm we don't test this method but in the future the query params can be different between search suggestion and a regular query